tests/LowLevelTests.hs

```diff
@@ -22,6 +22,7 @@ import           Data.List                                 (find)
 import qualified Data.Map.Strict                           as M
 import qualified Data.Set                                  as S
 import           GHC.Exts                                  (fromList, toList)
+import           Network.GRPC.Unsafe.ChannelArgs           (Arg(..))
```

Comment:
What was this for?

Reply:
The build of the test suite was failing due to one of the `Arg` constructors not being in scope.  I'm not sure why this wasn't an issue before

Reply:
Huh, weird. Well, thanks =)

Reply:
You're welcome!